### DISCUSSION TITLE
What is the Zip i need for cpu undervolting?

### DISCUSSION THREAD

**Discussion by Carlinho21** [seerge/g-helper#669] [General]
Hope im not being stupid but i cant seem to find the other zip i need for cpu undervolting. Installed 0.89 but i have no advanced tab. I see on the page for 0.89 it says Zip next to G-helper but i cant sem to find it.

Thanks

**Comment by seerge** [comment]
@Carlinho21 there is only 1 zip that you need, it's the one linked on home page https://github.com/seerge/g-helper/releases/latest/download/GHelper.zip

Undervolting works only for AMD CPUs (Ryzen 9 mainly). If you have Intel, you won't see advanced tab.

**Reply by Carlinho21** [reply]
Yer i just noticed, thanks dude. Sorry to be a pain haha
